package.json

```diff
@@ -77,7 +77,7 @@
         "coveralls": "^3.0.4",
         "crypto-js": "^3.1.9-1",
         "del": "^4.1.1",
-        "ethereumjs-wallet": "^0.6.3",
+        "ethers": "4.0.33",
```

Comment:
Putting quite some trust into the devs of `ether.js`. Also why is this version pinned here? The pinning should happen via `yarn.lock` or `package-lock.json` files. I think `^4.0.33` would be more appropriate.

Reply:
It's just a devDep, used to test that web3's wallet matches up with an independent implementation. `ethereumjs-wallet` was serving that purpose but it also depends on `scrypt.js` so `npm i` in the repo still resulted in `scrypt` package being installed and compiled.

Reply:
As for pinning vs. `^`, I've been bitten so many times by 3rd party patch bumps that violate semver (and have breaking changes) and/or bugs that I spec exact version deps without thinking about it. However, using `^` would be more consistent with what's spec'd for the other devDeps.